README.md

```diff
@@ -1,2 +1,4 @@
 # wecode-mentor
 A web app for Wecode community developers to meet and be paired for mentoring programs  
+
+# Added index.html
```

Comment:
We don't need to add what we have done here. this file is strictly for telling visitors what the project is about and how to use it.